README.md

```diff
@@ -18,7 +18,20 @@ gem 'cancancan-neo4j'
 ```
 
 ```ruby
-can :read, Article, author: { name: 'Chunky' }
+can :read, Article, author: { id: user.id }
 ```
 
-Here name is a property on Author and Article has 'has_one' relation with Author.
+Here `id` is a property on `Author` and `Article` has `has_one` relation with `Author`.
```

Comment:
It is atypical and confusing to have first code sample and afterwards an explanation. The explanation of what will be shown should come first and then the line of code. Have a look at other READMEs.